tools/gradle/commons/integrations/python.gradle

```diff
@@ -45,3 +58,6 @@ build.dependsOn installReqs
 check.dependsOn blackFormat
 check.dependsOn isortFormat
 check.dependsOn flakeCheck
+
+task pythonFormat(dependsOn: ['blackFormat', 'isortFormat', 'docFormat'])
```

Comment:
I like this. You can also replace the `check.dependsOn` calls with `check.dependsOn pythonFormat` and `check.dependsOn pythonLint`.